flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowDataSerDeSchemaTest.java

```diff
@@ -562,6 +578,55 @@ public void testSerializationMapNullKey() throws Exception {
         assertEquals(expectResult3, new String(actual3));
     }
 
+    @Test
+    public void testSerializationDecimalEncode() throws Exception {
+        String original =
+                "{\"decimal1\":123.456789,\"decimal2\":454621864049246170,\"decimal3\":0.000000027}";
+        String expectedWithPlainDecimal =
+                "{\"decimal1\":123.456789,\"decimal2\":454621864049246170,\"decimal3\":0.000000027}";
```

Comment:
1. They are equal? We may only need to declare just one `plainDecimalJson`?
2. Please move the variable near where it is used.